x-pack/legacy/plugins/ml/server/models/data_recognizer/data_recognizer.ts

```diff
@@ -822,21 +935,23 @@ export class DataRecognizer {
 
   // ensure the model memory limit for each job is not greater than
   // the max model memory setting for the cluster
-  async updateModelMemoryLimits(moduleConfig) {
+  async updateModelMemoryLimits(moduleConfig: Module) {
     const { limits } = await this.callWithRequest('ml.info');
     const maxMml = limits.max_model_memory_limit;
     if (maxMml !== undefined) {
-      const maxBytes = numeral(maxMml.toUpperCase()).value();
+      // @ts-ignore
+      const maxBytes: number = numeral(maxMml.toUpperCase()).value();
```

Comment:
Would be good to know what is being ignored here - to be consistent

Reply:
For some reason, the issue is `numeral` from `@elastic/numeral`. It complains that `value` prop does not exist.